source/io/Process.ooc

```diff
@@ -108,7 +108,11 @@ Process: abstract class {
 
 	// Returns a representation of the command, escaped to some point.
 	getCommandLine: func -> String {
-		args join(" ") replaceAll("\\", "\\\\")
+		result := ""
+		for (i in 0 .. args count - 1)
+			result = result << args[i] << " "
+		result = result << args[args count - 1]
```

Comment:
Previous code did nothing when `args` was empty. I think this will crash now.
I don't know if it is possible, though, but I wanted to point it out anyway.